crates/stackable-operator/src/builder/pod/container.rs

```diff
@@ -621,7 +622,9 @@ mod tests {
         assert!(ContainerBuilder::new("name_name").is_err());
         assert_container_builder_err(
             ContainerBuilder::new("name_name"),
-            "(e.g. \"example-label\", or \"1-label-1\", regex used for validation is \"[a-z0-9]([-a-z0-9]*[a-z0-9])?\")",
+            &format!(
+                "(e.g. \"example-label\", or \"1-label-1\", regex used for validation is \"{RFC_1123_LABEL_FMT}\""
+            ),
```

Comment:
Up to you, but I prefer using raw strings instead of escaping all of the double quotes:

```suggestion
            &format!(
                r#"(e.g. "example-label", or "1-label-1", regex used for validation is "{RFC_1123_LABEL_FMT}""#
            ),
```

Reply:
Same applies for the other `format!` above.

Reply:
https://github.com/stackabletech/operator-rs/pull/1064/commits/e2d062579dd304081dbee4fc221f254bdbf59124